FirebaseAnalyticsSwift/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Unreleased
+- Initial public beta release. Introduces new SwiftUI friendly APIs for
+  screen tracking. To use, add `pod 'FirebaseAnalyticsSwift'` to the Podfile or
```

Comment:
Will need a version specifier for the beta.  See the 7.0.0 release notes for an example